clientv3/cluster.go

```diff
@@ -131,6 +134,15 @@ func (c *cluster) MemberList(ctx context.Context) (*MemberListResponse, error) {
 	return nil, toErr(ctx, err)
 }
 
+func (c *cluster) MemberListLinearizable(ctx context.Context) (*MemberListResponse, error) {
```

Comment:
Question: do we want to add new API `MemberListLinearizable`? Or do we want to change the function signature of the existing API `MemberList` by adding an additional input parameter?

Reply:
Are we giving choice to user between current stale information and linearzable guarantee? Per the issue, thought we will always go with linearzable guarantee. I believe when you said changing the current signature with an additional input - you mean a bool for linearzable?

Reply:
+1 for making it default

/cc @xiang90 @jpbetz

Reply:
I like the idea of making linearizable the new default for `MemberList` call. Do we want to provide users the option (an input flag to `MemberList`) to get stale member list? (similar to serializable range request). Previously, users could get stale member list from a member that is isolated in the cluster.

Reply:
> I believe when you said changing the current signature with an additional input - you mean a bool for linearzable?

Yes.